pyartifactory/models/Auth.py

```diff
@@ -16,3 +16,11 @@ class ApiKeyModel(BaseModel):
 
 class PasswordModel(BaseModel):
     password: SecretStr
+
+
+class AccessTokenModel(BaseModel):
+    access_token: str
+    expires_in: Optional[int] = 0

```

Comment:
Could you please set this variable to 3600

Reply:
done

Reply:
:+1: